data/Tonga/Assembly/sources/instructions.json

```diff
@@ -4,13 +4,31 @@
       "file": "morph/data.csv",
       "create": {
         "from": "morph",
-        "scraper": "duncanparkes/tonga",
-        "query": "SELECT id, name, constituency, 'unknown' AS party, term_id AS term, image, email, phone, cell, details_url AS source FROM data"
+        "scraper": "everypolitician-scrapers/tonga-legislative-assembly",
+        "query": "SELECT id, name, constituency, 'unknown' AS party, 2015 AS term, image, email, phone, cell, source FROM data"
       },
       "source": "http://parliament.gov.to/",
       "type": "membership"
     },
     {
+      "file": "morph/wikidata-members.csv",
+      "create": {
+        "from": "morph",
+        "scraper": "everypolitician-scrapers/tonga-wikidata-members",
+        "query": "SELECT item as id, itemlabel as name, itemaltlabel as other_name, date(start) as start_date, date(end) AS end_date, email, 'unknown' AS party, 2015 AS term FROM data"
+      },
+      "type": "membership"
+    },
+    {
+      "file": "archive/data-changed.csv",
+      "type": "person",
```

Comment:
Removed the `memberships` fields in https://github.com/everypolitician/everypolitician-data/pull/24902/commits/bb0698f2097a7127d8645173031455d0bf1b6497